packages/wonder-blocks-tooltip/src/components/tooltip-popper.tsx

```diff
@@ -64,11 +78,42 @@ const filterPopperPlacement = (
     }
 };
 
+type SmallViewportModifier = Modifier<"smallViewport", object>;
```

Comment:
I've never seen `object` used like this before. Is this basically the `any` of objects?